### DISCUSSION TITLE
Any way to debug tauri constantly reloading the client?

### DISCUSSION THREAD

**Discussion by grayfallstown** [tauri-apps/tauri#5948] [Q&A] [answered]
Tauri is constantly sending reload events via websockets. Without making code changed, just calling `cargo tauri dev` the wevview reloads 3 to 4 times and then randomly.

What is the recommended strategy to debug this?
Can we disable this reload functionality altogether?

**Comment by JonasKruckenberg** [comment, marked as answer]
Handling reloading during development (and HMR too) is out if scope of Tauri and instead is handled by the frontend framework/stack you chose. In most cases this will be vite.

(The only case where Tauri handles refresh logic is when you configure it to not use any dev server but point it to a static file instead)

So to fix any refresh issue you'll likely have to look to your dev-server for options

**Comment by grayfallstown** [comment]
Thanks for the hint. I moved my index.html and style.css to a new folder called `static`, changed my tauri.conf.json `build.devPath` and `build.distDir` from `./` to `./static` and the issue is gone.
